compiler+runtime/src/cpp/clojure/string_native.cpp

```diff
@@ -0,0 +1,94 @@
+#include <boost/algorithm/string.hpp>
+
+#include <fmt/format.h>
+
+#include <jank/native_persistent_string/fmt.hpp>
+#include <clojure/core_native.hpp>
+#include <jank/runtime/convert.hpp>
+#include <jank/runtime/core.hpp>
+#include <jank/runtime/context.hpp>
+#include <jank/runtime/behavior/callable.hpp>
+#include <jank/runtime/visit.hpp>
```

Comment:
You're not using any visting fns, while this is an incredibly expensive include. Please audit all of the includes here to ensure they're needed.

Reply:
Still working on this.